### PR TITLE
Add simplified Chinese js translation

### PR DESCRIPTION
**Pull request by Artoria2e5**
This single-file commit, heavily based on the zh-hk version as of
ec9999c, fixes the missing js locale. Intentional, significant
departures from current zh-cn yml localizations have been marked with
rationales given.

### CODE REVIEW

**Review by Gargron — APPROVED** [review]

File: app/assets/javascripts/components/locales/zh-cn.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  "account.unfollow": "取消关注",
+  "account.unmute": "取消 @{name} 的静音",
+  "boost_modal.combo": "如你想在下次路过时显示，请按{combo}，",
+  "column_back_button.label": "返回",
+  "column.blocks": "屏蔽用户",
+  "column.community": "本站时间轴",
+  // intentional departure from existing "推文" translation for posts:
+  // "推文" refers to "推特", the official translation for Twitter.
+  // Currently using a semi-phonetic translation "嘟", which refers
+  // to train horn sounds, for "toot".
```
Comment (Artoria2e5) [inline app/assets/javascripts/components/locales/zh-cn.jsx]: I should try to read some children's books and figure out how people refer to elephant horn sounds in Chinese....

File: app/assets/javascripts/components/locales/zh-cn.jsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  "empty_column.home": "你还没有关注任何用户。快看看{public}，向其他用户搭讪吧。",
+  "empty_column.notifications": "你没有任何通知纪录，快向其他用户搭讪吧。",
+  "empty_column.public": "跨站公共时间轴暂时没有内容！快写一些公共的嘟文，或者关注另一些服务站的用户吧！你和本站、友站的交流，将决定这里出现的内容。",
+  "follow_request.authorize": "批准",
+  "follow_request.reject": "拒绝",
+  "getting_started.about_addressing": "只要你知道一位用户的用户名称和域名，你可以用“@用户名称@域名”的格式在搜索栏寻找该用户。",
+  "getting_started.about_shortcuts": "只要该用户是在你现在的服务站开立，你可以直接输入用户𠱷搜索。同样的规则适用于在嘟文提及别的用户。",
+  "getting_started.apps": "手机或桌面应用程序",
+  "getting_started.heading": "开始使用",
+  "getting_started.open_source_notice": "Mastodon 是一个开放源码的软件。你可以在官方 GitHub ({github}) 贡献或者回报问题。你亦可透过{apps}阅读 Mastodon 上的消息。",
```
Comment (gumblex) [inline app/assets/javascripts/components/locales/zh-cn.jsx]: 透过

File: app/assets/javascripts/components/locales/zh-cn.jsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  "empty_column.hashtag": "这个标签暂时未有内容。",
+  "empty_column.home": "你还没有关注任何用户。快看看{public}，向其他用户搭讪吧。",
+  "empty_column.home.public_timeline": "公共时间轴",
+  "empty_column.home": "你还没有关注任何用户。快看看{public}，向其他用户搭讪吧。",
+  "empty_column.notifications": "你没有任何通知纪录，快向其他用户搭讪吧。",
+  "empty_column.public": "跨站公共时间轴暂时没有内容！快写一些公共的嘟文，或者关注另一些服务站的用户吧！你和本站、友站的交流，将决定这里出现的内容。",
+  "follow_request.authorize": "批准",
+  "follow_request.reject": "拒绝",
+  "getting_started.about_addressing": "只要你知道一位用户的用户名称和域名，你可以用“@用户名称@域名”的格式在搜索栏寻找该用户。",
+  "getting_started.about_shortcuts": "只要该用户是在你现在的服务站开立，你可以直接输入用户𠱷搜索。同样的规则适用于在嘟文提及别的用户。",
```
Comment (gumblex) [inline app/assets/javascripts/components/locales/zh-cn.jsx]: 用户𠱷

File: app/assets/javascripts/components/locales/zh-cn.jsx (hunk 2 above)
Comment (Artoria2e5) [inline app/assets/javascripts/components/locales/zh-cn.jsx]: Ugh, will fix.

**Pull request merged**